scripts/globals/bluemagic.lua

```diff
@@ -349,6 +349,9 @@ xi.spells.blue.useDrainSpell = function(caster, target, spell, params, softCap,
     dmg = addBonuses(caster, spell, target, dmg)
     dmg = adjustForTarget(target, dmg, spell:getElement())
 
+    -- Check if the mob has a damage cap
+    target:checkDamageCap(dmg)
```

Comment:
This does nothing?

Reply:
I accidently deleted the dmg = when I changed the name over and move it below adjustments. Fixed.